sdk/database/helper/connutil/cloudsql.go

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package connutil
+
+import (
+	"fmt"
+
+	"cloud.google.com/go/cloudsqlconn"
+	"cloud.google.com/go/cloudsqlconn/postgres/pgxv4"
+	"cloud.google.com/go/cloudsqlconn/sqlserver/mssql"
+)
+
+func (c *SQLConnectionProducer) getCloudSQLDriverName() (string, error) {
+	var driverName string
+	switch c.Type {
+	case dbTypeMSSQL:
```

Comment:
The MSSQL cases will be removed from this file. GCP IAM only supports Postgres and MySQL